central/reports/scheduler/schedule.go

```diff
@@ -486,6 +513,15 @@ func (s *scheduler) getDeploymentIDs(ctx context.Context, deploymentsQuery *v1.Q
 	return search.ResultsToIDs(results), nil
 }
 
+func (s *scheduler) Start() {
+	go s.runReports()
+}
+
+func (s *scheduler) Stop() {
+	s.stopper.Client().Stop()
+	_ = s.stopper.Client().Stopped().Wait()
```

Comment:
Something regarding the original implementation: don't ignore the error, at least log it.